inference-engine/src/vpu/graph_transformer/src/middleend/passes/weights_analysis.cpp

```diff
@@ -208,11 +208,11 @@ void PassImpl::run(const Model& model) {
 
     static const int scaleThreshold = 1;
 
-    bool isGrowingOutput = checkGrowingOutput(model);
+    const auto& env = CompileEnv::get();
+    bool isGrowingOutput = env.config.checkPreprocessingInsideModel ? checkGrowingOutput(model) : false;
```

Comment:
Nitpick: looks a bit strange - the fact if output is growing does not depend on your option - I think it's more clear to check your new option at the beginning of the method and return if it's false

Reply:
Or it's even better not to put a pass into pass manager in case if options is not set

Reply:
We need to perform scaling no matter this option is set or not.
Moved this to the beginning of `checkGrowingOutput`.

Reply:
If we need to perform scaling regardless of this option - what this option stands for?

Reply:
You may refer to the description of this option. This option is used to adjust scaling algorithm, but not to enable/disable it.

Reply:
Description of the option does not say what option does. It says when we need to set "NO" to its value

Reply:
Yeah, it doesn't tell us directly, but I thought it can be deduced from the description and option's name.
Can this be resolved?